spec/receipt_printer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'receipt_printer'
+
+describe ReceiptPrinter do
+  subject(:receipt_printer) {described_class.new(restaurant_data, totals)}
+  let(:restaurant_data) {{:shopname => :the_coffee_connection,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.